lib/3scale_toolbox/commands/application_command/create_command.rb

```diff
@@ -0,0 +1,110 @@
+module ThreeScaleToolbox
+  module Commands
+    module ApplicationCommand
+      module Create
+        class CreateSubcommand < Cri::CommandRunner
+          include ThreeScaleToolbox::Command
+
+          def self.command
+            Cri::Command.define do
+              name        'create'
+              usage       'create [opts] <remote> <account> <service> <application-plan> <name>'
+              summary     'create one application'
+              description 'create one application linked to given account and application plan'
+
+              option      nil, 'user-key', 'User Key (API Key) of the application to be created.', argument: :required
+              option      nil, 'application-id', 'App ID or Client ID (for OAuth and OpenID Connect authentication modes) of the application to be created. ', argument: :required
```

Comment:
same thing about the symbol as in other commands

Reply:
not necessarily